new_course/1_programming_fundementals/methods.md

```diff
@@ -73,27 +73,27 @@ Your method names can use numbers, capital letters, lowercase letters, and the s
 
 Here are some things you are not allowed to do with your method names:
 
-* You cannot name your method one of Ruby's approximately 40 reserved words, such `end`, `while`, or `for`. Do a quick scan of the [full list here](http://www.java2s.com/Code/Ruby/Language-Basics/Rubysreservedwords.htm).
+* You cannot name your method one of Ruby's approximately 40 reserved words, such as `end`, `while`, or `for`. Checkout the full list [here](http://www.java2s.com/Code/Ruby/Language-Basics/Rubysreservedwords.htm).
 * You cannot use any symbols other than `_`, `?`, `!`, and `=`.
 * You cannot use `?`, `!`, or `=` anywhere other than at the end of the name.
 * You cannot begin a method name with a number.
 
 Here are some examples of valid and invalid method names:
 
-~~~ruby
-method_name     # valid
-_name_of_method # valid
-1_method_name    # invalid
-method_27       # valid
-method?_name    # invalid
-method_name!    # valid
-begin           # invalid  (Ruby reserved word)
-begin_count     # valid
+~~~
+method_name      # valid
+_name_of_method  # valid
+1_method_name    # invalid (Starts with a number)
+method_27        # valid
+method?_name     # invalid (Valid symbols. such as ?, can only be used at the end of the name)
```

Comment:
Should read:

method?_name     # invalid (Valid symbol, such as "?", can only be used at the end of the name)